.github/workflows/issue-labeler.yml

```
@@ -0,0 +1,35 @@
+name: Issue labeler
```

Best practice to align workflow name with filename.

```suggestion
name: issue-labeler
```

---

updated